contracts/adapters/DPMAdapter.sol

```diff
@@ -48,7 +43,7 @@ contract DPMAdapter is ISecurityAdapter {
         return accountGuard.canCall(proxyAddress, operator) || (operator == positionOwner);
     }
 
-    function permit(bytes memory triggerData, address target, bool allowance) public onlyDelegate {
+    function permit(bytes memory triggerData, address target, bool allowance) public {
```

Comment:
can be called directly to permit any address

Reply:
https://github.com/OasisDEX/automation-smartcontracts/pull/186/commits/8d84a77c68b41a4039aba7e36f68e8f7f89d1018